website/docs/enterprise/user-permissions.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: User permissions
+sidebar_position: 1
+hide_title: true
+---
+import TierLabel from "../_components/TierLabel";
+
+# User permissions
+
+Weave GitOps Enterprise extends Weave GitOps [permissions](../configuration/user-permissions.mdx) by adding more roles. These roles may need to be extended further in order to support certain use cases. Some of the most common use cases are described below.
+
+## Progressive delivery with Flagger
+
+Weave GitOps Enterprise integrates with Flagger in order to provide a view on progressive delivery deployments. This includes the ability to view all the resources that Flagger manages during its operation. The default ClusterRole `gitops-canaries-reader` includes the minimum permissions necessary for a user to be able to view canary object details, metric template object details and canary related events. 
+
+When Flagger is configured to integrate with a service mesh such as Linkerd or Istio for the rollout, then this ClusterRole needs to be extended so that it can read the additional service mesh resources being generated by Flagger. Note that currently, in order to display service mesh or ingress related resources, we require `spec.provider` to be set in each canary resource. 
+
+The following table provides a list of all the custom resources that Flagger generates grouped by provider:
+
+| Provider          | API Group                 | Resource        |
```

Comment:
non-blocking - but is there a source of truth for this? Whilst useful my only concern is whether it stays current as flagger or the technologies it is compatible with evolve.

Reply:
Not in this table form, this was extracted by looking into Flagger source code i.e. https://github.com/fluxcd/flagger/tree/main/pkg/apis which contains the list of the different integration types used.